app/pb/communities.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package org.couchers.api.communities;
+
+import "google/protobuf/timestamp.proto";
+
+import "pb/api.proto";
+
+message Point {
+  double lat = 1;
+  double lng = 2;
+}
+
+message Node {
+  int32 node_id = 1;
+  string name = 2;
+  org.couchers.api.core.NullableInt32Value parent_node = 3;
```

Comment:
I'm not sure if the this needs to be nullable here. We could use `0` (the default if no int32 is specified) to denote no parent.

Reason being that these (nullablesXXvalues) are a bit annoying to work with, and there should only be one node with no parent.